packages/ui/src/message-room-details-dialog/components/EditDescriptionModal.jsx

```diff
@@ -11,7 +12,45 @@ import {
   ModalTopic
 } from "../MessageRoomDetailsDialog.styled";
 
-const EditDescriptionModal = ({ closeEdit }) => {
+const EditDescriptionModal = ({ closeEdit, roomData }) => {
+  const roomDescription = roomData.data.description;
+  const descriptionRef = useRef();
+
+  // to update data
+  const handleDesc = () => {
+    const org_id = roomData.data.org_id;
+    const member_id = roomData.data.created_by;
+    const room_id = roomData.data._id;
+
+    const editedDescription = {
+      room_name: roomData.data.room_name,
+      description: descriptionRef.current.value,
+      topic: roomData.data.topic,
+      is_private: roomData.data.is_private,
+      is_archived: roomData.data.is_archived
+    };
+
+    fetch(
```

Comment:
also why are you using fetch instead of axios that's already installed?

Reply:
I'll change that now